packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -4706,6 +4706,135 @@ class Wrap extends MultiChildRenderObjectWidget {
 ///  * [CustomMultiChildLayout], which uses a delegate to position multiple
 ///    children.
 ///  * The [catalog of layout widgets](https://flutter.dev/widgets/layout/).
+///
+/// {@tool snippet --template=freeform}
+///
+/// This example uses the [Flow] widget to create a menu that opens and closes
+/// as it is interacted with. The [FlowDelegate] uses the [AnimationController]
+/// to paint the children of the [Flow].
+///
+/// ![A menu created using the Flow widget opens and closes.](https://flutter.github.io/assets-for-api-docs/assets/widgets/flow_menu.mp4)
```

Comment:
When the menu opens, the house icon has a blue background and the background if the exclamation mark is amber. When the menu closes, the house becomes amber and the exclamation mark blue. It looks a little strange. Is that expected?

Reply:
Oh, i see. The amber icon is the "last selected" icon - too bad you can't see the tapping finger in the video... Maybe add this to the example description above?

Reply:
I can make that clearer. And perhaps add an InkWell to the animation to better illustrate it too. I will revisit and address your other comments as well. Thanks!